packages/react-components/src/components/RichTextEditor/RichTextSendBox.tsx

```diff
@@ -438,26 +438,16 @@ export const RichTextSendBox = (props: RichTextSendBoxProps): JSX.Element => {
     onSendMessage
   ]);
 
-  const hasErrorMessage = useMemo(() => {
+  const hasErrorMessageThatPreventsSubmit = useMemo(() => {
     return (
-      !!systemMessage ||
       !!contentTooLongMessage ||
       /* @conditional-compile-remove(file-sharing-acs) */
-      !!attachmentUploadsPendingError ||
-      /* @conditional-compile-remove(file-sharing-acs) */
-      !!attachments?.filter((attachmentUpload) => attachmentUpload.error).pop()?.error ||
-      /* @conditional-compile-remove(rich-text-editor-image-upload) */
-      !!inlineImages?.filter((image) => image.error).pop()?.error
+      !!attachmentUploadsPendingError
```

Comment:
Could you add comments why it works in this way? (that we don't include inline image error)